### DISCUSSION TITLE
Button Label showing HTML Character instead of apostrophe

### DISCUSSION THREAD

**Discussion by 7met** [wireui/wireui#641] [General]
I am having an issue with with the x-button component when passing in a label. The value I am passing from Livewire is `Braggard's Pain` but what is getting displayed is `Braggard&#039;s Pain`, how can I pass an apostrophe without it being converted?

**Comment by ph7jack** [comment]
@7met Can you show me how you are passing the variable to the component?

**Comment by 7met** [comment]
Sure thing, here is the code

```
<x-button class="w-full" label="{{ $macro->label }}" wire:click="triggerMacro('{{ $macro->external_id }}')"/>
```

**Reply by ph7jack** [reply]
In this way, you will encode twice the label.
See the solution
`<x-button class="w-full" :label="$macro->label" wire:click="triggerMacro('{{ $macro->external_id }}')" />`

The `{{ }}` will encode the label and pass it to the component. 
The component will encode again the label using `{{ }}` .
To prevent this we just need to pass the full variable to the blade component directly.

**Comment by 7met** [comment]
Oh yeah, sorry I see it now. Thanks again for the help!

--
Sean Shoffstall
***@***.***
415-850-9619
